### PR TITLE
enabled auth and tls for pravega adaptor

### PR DESCRIPTION
**Pull request by vangork**
1. change connection_str to accept controller_uri (tcp://addr:port or tls://addr:port).
2. put fixed-routing-key and keycloak-file property setting in the config file.
```
[message-broker]
keycloak-file = /home/ubuntu/keycloak1.json
fixed-routing-key = foo
```

to verify:
checkout https://github.com/pravega/gstreamer-pravega/tree/NPS-8613-rewrite-integration-test in the deepsteam pod.
```
cd ./gstreamer-pravega/deepstream/pravega_protocol_adapter 
make test
```

### CODE REVIEW

**Review by claudiofahey — APPROVED** [review]
Looks good. When we start deploying in Kubernetes, we may find it more convenient to pass dynamic properties via URL instead of the config file. We'll decide that later. For now, we will use this approach which is quite standard for DeepStream.
I'll merge this after https://github.com/pravega/gstreamer-pravega/pull/28.

**Comment by claudiofahey** [comment]
@vangork, please rebase this onto the latest master.

**Comment by vangork** [comment]
> @vangork, please rebase this onto the latest master.

done.

File: Cargo.lock [hunk 1]
```diff
@@ -150,7 +150,7 @@ dependencies = [
 [[package]]
 name = "atk"
 version = "0.13.0"
-source = "git+https://github.com/gtk-rs/gtk-rs#18bafda64a6d8cd5b4e23c2053543853f75ed7b7"
```
Comment (claudiofahey) [inline Cargo.lock:153]: We need to avoid changing the version of anything related to gstreamer, gtk, and glib. These often have breaking changes. When adding dependencies to Cargo.toml, it will want to update to the latest version. You'll then need to manually edit Cargo.lock and search/replace commit hashes to undo the changes.

**Review by claudiofahey — CHANGES_REQUESTED** [review]

File: Cargo.lock (hunk 1 above)
Comment (vangork) [reply Cargo.lock]: revert back

**Review by claudiofahey — APPROVED** [review]
